libethcore/Transaction.h

```diff
@@ -142,6 +142,8 @@ class TransactionBase
 	/// @returns the signature of the transaction. Encodes the sender.
 	SignatureStruct const& signature() const { return m_vrs; }
 
+	bool isForwardingCall() const { return m_vrs.zeroSign(); }
```

Comment:
Not sure if this should be called a forwarding call. Perhaps just `hasZeroSignature`?